Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,23 +1,174 @@
 import React from 'react';
 import './App.css';
+import Display from './components/DisplayComponents/CalculatorDisplay';
+import ButtonContainer from './components/ButtonComponents/ButtonContainer';
 
-const App = () => {
-  return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
-    </div>
-  );
-};
+const buttons = [
+  {
+      id: '0',
+      name: '0',
+      style: 'triple',
+      type: 'number'
+  },
+  {
+      id: '1',
+      name: '1',
+      style: 'single',
+      type: 'number'
+  },
+  {
+      id: '2',
+      name: '2',
+      style: 'single',
+      type: 'number'
+  },
+  {
+      id: '3',
+      name: '3',
+      style: 'single',
+      type: 'number'
+  },
+  {
+      id: '4',
+      name: '4',
+      style: 'single',
+      type: 'number'
+  },
+  {
+      id: '5',
+      name: '5',
+      style: 'single',
+      type: 'number'
+  },
+  {
+      id: '6',
+      name: '6',
+      style: 'single',
+      type: 'number'
+  },
+  {
+      id: '7',
+      name: '7',
+      style: 'single',
+      type: 'number'
+  },
+  {
+      id: '8',
+      name: '8',
+      style: 'single',
+      type: 'number'
+  },
+  {
+      id: '9',
+      name: '9',
+      style: 'single',
+      type: 'number'
+  },
+  {
+      id: '10',
+      name: 'clear',
+      style: 'triple',
+      type: 'clear'
+  },
+  {
+      id: '1',
+      name: '÷',
+      style: 'action',
+      type: 'action'
+  },
+  {
+      id: '2',
+      name: 'x',
+      style: 'action',
+      type: 'action'
+  },
+  {
+      id: '3',
+      name: '-',
+      style: 'action',
+      type: 'action'
+  },
+  {
+      id: '4',
+      name: '+',
+      style: 'action',
+      type: 'action'
+  },
+  {
+      id: '5',
+      name: '=',
+      style: 'action',
+      type: 'action'
+  }
+]
```

Comment:
While it may work in this scenario, I think your ids in a single array of objects should always be unique within that array, otherwise this type of design will cause issues down the road